driver/wnbd.inf

```diff
@@ -50,8 +51,8 @@ LoadOrderGroup = SCSI Miniport
 AddReg         = pnpsafe_isa_addreg
 
 [pnpsafe_isa_addreg]
-HKR,,"BreakOnEntry", %REG_DWORD%,  0x00000000
-HKR,"Parameters","BusType",%REG_DWORD%,0x0000000E
+HKR, "Parameters\PnpInterface", "5", %REG_DWORD%, 0x00000001
```

Comment:
Ok, so we're using "5" because it's a PCI bus.

Reply:
Ooops.
Do you want me to add it to the commit message?